app/assets/javascripts/discourse/tests/fixtures/discovery-fixtures.js

```diff
@@ -129,7 +129,7 @@ export default {
       { id: 6680, username: "cdman", avatar_template: "/images/avatar.png" },
       { id: 500, username: "aeid", avatar_template: "/images/avatar.png" },
       { id: 8, username: "geek", avatar_template: "/images/avatar.png" },
-      { id: 606, username: "Cafeine", avatar_template: "/images/avatar.png" }
+      { id: 606, username: "Caffeine", avatar_template: "/images/avatar.png" }
```

Comment:
https://github.com/discourse/discourse/pull/12779#discussion_r617131932
I'm assuming that these usernames aren't particularly important. I'm happy to drop any changes.

For the most part, I've actually skipped usernames...